ocp_resources/datavolume.py

```diff
@@ -172,6 +202,12 @@ def to_dict(self):
                 }
             if self.preallocation is not None:
                 self.res["spec"]["preallocation"] = self.preallocation
+            if self.delete_after_completion is not None:
```

Comment:
```suggestion
            if self.delete_after_completion:
```
`delete_after_completion` is an str

Reply:
all the fields that are not mandatory are set to None here, most of them are strings. should I change it?

Reply:
depends on the use case; is an empty string, for example, acceptable?

Reply:
we should not force/check the user input, the docstring should state that "true" and "false" are acceptable inputs

Reply:
@dalia-frank please address this comment and modify the code

Reply:
@rnetser already modified
`if self.delete_after_completion:`